process/elasticproc/transactions/scrsDataToTransactions.go

```diff
@@ -65,127 +47,35 @@ func (st *scrsDataToTransactions) processTransactionsAfterSCRsWereAttached(trans
-		return
-	}
-
-	if st.hasSCRWithErrorCode(tx) {
-		tx.Status = transaction.TxStatusFail.String()
-	}
-}
-
-func (st *scrsDataToTransactions) hasSCRWithErrorCode(tx *data.Transaction) bool {
-	for _, scr := range tx.SmartContractResults {
-		for _, codeStr := range st.retCodes {
-			if strings.Contains(string(scr.Data), hex.EncodeToString([]byte(codeStr))) ||
-				scr.ReturnMessage == codeStr {
-				return true
-			}
-		}
-	}
-
-	return false
-}
-
-func hasSuccessfulSCRs(tx *data.Transaction) bool {
-	for _, scr := range tx.SmartContractResults {
-		if isScResultSuccessful(scr.Data) {
-			return true
-		}
-	}
-
-	return false
 }
 
-func hasCrossShardPendingTransfer(tx *data.Transaction) bool {
-	for _, scr := range tx.SmartContractResults {
-		splitData := strings.Split(string(scr.Data), data.AtSeparator)
-		if len(splitData) < 2 {
-			continue
-		}
-
-		isMultiTransferOrNFTTransfer := splitData[0] == core.BuiltInFunctionESDTNFTTransfer || splitData[0] == core.BuiltInFunctionMultiESDTNFTTransfer
-		if !isMultiTransferOrNFTTransfer {
-			continue
-		}
-
-		if scr.SenderShard != scr.ReceiverShard {
-			return true
-		}
-	}
-
-	return false
-}
-
-func (st *scrsDataToTransactions) processSCRsWithoutTx(scrs []*data.ScResult) (map[string]string, map[string]*data.FeeData) {
-	txHashStatus := make(map[string]string)
+func (st *scrsDataToTransactions) processSCRsWithoutTx(scrs []*data.ScResult) map[string]*data.FeeData {
 	txHashRefund := make(map[string]*data.FeeData)
 	for _, scr := range scrs {
-		if scr.InitialTxGasUsed != 0 {
-			var feeNum float64
-			var err error
-			initialTxFeeBig, ok := big.NewInt(0).SetString(scr.InitialTxFee, 10)
-			if ok {
-				feeNum, err = st.balanceConverter.ComputeESDTBalanceAsFloat(initialTxFeeBig)
-			}
-			if err != nil {
-				log.Warn("scrsDataToTransactions.processSCRsWithoutTx: cannot compute fee as num",
-					"initial Tx fee", initialTxFeeBig, "error", err)
-			}
-
-			txHashRefund[scr.OriginalTxHash] = &data.FeeData{
-				FeeNum:   feeNum,
-				Fee:      scr.InitialTxFee,
-				GasUsed:  scr.InitialTxGasUsed,
-				Receiver: scr.Receiver,
-			}
-		}
-
-		if !st.isESDTNFTTransferOrMultiTransferWithError(string(scr.Data)) {
+		if scr.InitialTxGasUsed == 0 {
 			continue
 		}
 
-		txHashStatus[scr.OriginalTxHash] = transaction.TxStatusFail.String()
-	}
-
-	return txHashStatus, txHashRefund
-}
-
-func (st *scrsDataToTransactions) isESDTNFTTransferOrMultiTransferWithError(scrData string) bool {
-	splitData := strings.Split(scrData, data.AtSeparator)
-	isMultiTransferOrNFTTransfer := splitData[0] == core.BuiltInFunctionESDTNFTTransfer || splitData[0] == core.BuiltInFunctionMultiESDTNFTTransfer
-	if !isMultiTransferOrNFTTransfer || len(splitData) < minNumOfArgumentsNFTTransferORMultiTransfer {
-		return false
-	}
+		var feeNum float64
+		var err error
+		initialTxFeeBig, ok := big.NewInt(0).SetString(scr.InitialTxFee, 10)
+		if ok {
+			feeNum, err = st.balanceConverter.ComputeESDTBalanceAsFloat(initialTxFeeBig)
```

Comment:
is `ComputeESDTBalanceAsFloat` the right function to use? If so, it could use a renaming